### DISCUSSION TITLE
Github 是全球最大的同性交友平台？

### DISCUSSION THREAD

**Discussion by JSHT** [1101-GS4719-C/Python-Community-GS4719-C#611] [優秀專案]
# [Dress](https://github.com/komeiji-satori/Dress)
這個有趣的專案到目前為止總計收穫了 21.2k 星星，是一個女裝分享 Repo，你可以透過發 PR 的方式分享自己的女裝，也可以透過其架設的網站進行上傳。
果然自己的女朋友自己扮從來都不是都市傳說。

**Comment by chia-kai-chang** [comment]
好酷，我們也應該來玩一下，讓大家使用一下 PR 功能~

**Comment by awkward-willy** [comment]
俗說話的好：女裝只有0次和無限次的區別
在此預祝各位女裝快樂，但我先逃跑了

**Reply by JSHT** [reply]
可以拍朋友女裝😂😂

**Comment by HsuChen88** [comment]
GitHub 是世界最大的女裝儲存庫

**Comment by JSHT** [comment]
剛想到可以透過 [Disco Diffusion](https://colab.research.google.com/github/alembics/disco-diffusion/blob/main/Disco_Diffusion.ipynb) 畫畫看女裝，結果如下圖。看來這個東西還是比較適合畫大景，不適合畫人物。(或是我提供的資訊不夠多、具體，我輸入的是 "A pretty drag queen with dress", "animate")
![dragQueen(0)_0](https://user-images.githubusercontent.com/48956859/169751097-0c68b5f7-23e5-4a35-b644-9af17c9d9d1f.png)
![dragQueen(0)_1](https://user-images.githubusercontent.com/48956859/169751104-05656d01-c111-4458-966e-ed424f8e1cbe.png)
希望大家一起來試試看，可以參考 [空罐王](https://www.youtube.com/watch?v=w7B1rfhnqY8) 的介紹使用。

**Reply by JSHT** [reply]
另外也想問有人拿到他們團隊開發的 Midjourney 測試資格嗎? 好奇跑出來會怎樣

**Comment by minniewei** [comment]
哼哼，期待大家的測試結果，說不定會發現新世界呦

**Comment by Lai27330** [comment]
想不到這個感覺非常專業的學術交流平台上還有這麼有趣的內容，看起來突然變得親民一點的感覺了XD。

**Comment by jason79461385** [comment]
摁，突然覺得自己點進了不該點的東西

**Comment by yucheng1028** [comment]
9ay警報

**Comment by yenwei12138** [comment]
看標題我還以為是指Github上大多都是男性，殊不知是女裝大佬的部分......

**Comment by liangyinglly** [comment]
原來使用github的不只是各方程設大大

**Comment by etin2468** [comment]
怕，我先不要

**Comment by Lynn12378** [comment]
上面的圖片有嚇到我哈哈
